javascript-typing-tutor/main.js

```diff
@@ -0,0 +1,14 @@
+var current = document.querySelectorAll("span");
+var i = 0;
+var counter = 0;
+var results = (100 / counter);
+function typedLetter() {
+  var key = event.key;
+    if (key == current[i].textContent) {
```

Comment:
You should be using strict equality (`===`) here, so that you're making sure the type matches as well